robottelo/ui/domain.py

```diff
@@ -1,7 +1,9 @@
 # -*- encoding: utf-8 -*-
 """Implements Domain UI"""
+from robottelo.constants import FILTER
 from robottelo.ui.base import Base
 from robottelo.ui.locators import common_locators, locators
```

Comment:
that's fine to have:
`from robottelo.ui.locators import common_locators, locators, tab_locators`